pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "heal"
```

Comment:
```suggestion
name = "heal-sdk"
```
i think we should rename this so when publishing, the package name would be `heal-sdk`

Reply:
Excellent! Thank you for catching that!